implementations/rust/ockam/ockam_api/src/config/cli.rs

```diff
@@ -140,3 +143,35 @@ impl ConfigValues for StartupConfig {
         Self::default()
     }
 }
+
+#[derive(Clone, Debug, Default, Serialize, Deserialize)]
+pub struct AuthoritiesConfig {
+    authorities: BTreeMap<Vec<u8>, MultiAddr>,
```

Comment:
That is fine, but for small number of elements it's actually makes sense to use `Vec` (faster, simpler, smaller memory footprint)